src/AzureDataFactory.TestingFramework/AzureDataFactory.TestingFramework.csproj

```diff
@@ -11,12 +11,17 @@
       <PackageReference Include="Azure.Core.Expressions.DataFactory" Version="1.0.0-beta.4" />
       <PackageReference Include="Azure.ResourceManager" Version="1.7.0" />
     <PackageReference Include="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20230804.1" PrivateAssets="All" />
-
     </ItemGroup>
 
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>AzureDataFactory.TestingFramework.Tests</_Parameter1>
+        </AssemblyAttribute>
+    </ItemGroup>
+    
     <PropertyGroup>
         <PackageId>AzureDataFactory.TestingFramework</PackageId>
-        <PackageVersion>0.1.6-alpha</PackageVersion>
+        <PackageVersion>0.1.7-alpha</PackageVersion>
```

Comment:
We need to document the version evolution in the main ReadMe and document the differences (weather if its the ADF underlying version that changes and/or the framework itself)

Reply:
Great idea, I'll add a separate issue to introduce a changelog.